lib/Cleantalk/Common/API.php

```diff
@@ -765,7 +765,45 @@ public static function methodSendLocalSettings(
 
         return static::sendRequest($request, self::URL, 0);
     }
-
+    
+    /**
+     * Call check_bot API method
+     *
+     * Make a decision if it's bot or not based on limited input JavaScript data
+     *
+     * @param $api_key               string      Access key
+     * @param $event_token           string|null Unique event ID
+     * @param $event_javascript_data string      JSON-string params to analysis
+     * @param $browser_sign          string      Browser ID
+     * @param $sender_ip             string      IP address
+     * @param $event_type            string      'GENERAL_BOT_CHECK' || 'CONTACT_DECODING'
+     * @param $message_to_log        string      Custom message
+     *
+     * @return array|bool
+     */
+    public static function methodCheckBot(
```

Comment:
The `check_bot` method is on moderate servers, not on api servers!

Reply:
This is big... If add properly, the Cleantalk class should be refactored. Will try to avoid unnecessary refactoring.